Tests/TensorFlowTests/OperatorTests/LinearAlgebraTests.swift

```diff
@@ -84,11 +84,31 @@ final class LinearAlgebraTests: XCTestCase {
         let expectedGradient = Tensor([a, b])
         assertEqual(computedGradient, expectedGradient, accuracy: 1e-16)
     }
+
+    func testLogdet() {
+        let input = Tensor<Float>([[[6.0, 4.0], [4.0, 6.0]], [[2.0, 6.0], [6.0, 20.0]]])
+        let expected = Tensor<Float>([2.9957323, 1.3862934])
+        let computed = logdet(input)
+        assertEqual(computed, expected, accuracy: 1e-5)
+    }
     
+    func testLogdetGradient() {
```

Comment:
Thanks for adding tests!

Could you please add a Python TensorFlow test comment? You can adapt the one from `cholesky`:
```swift
        // import tensorflow as tf
        // x = tf.constant([[[6., 4.], [4., 6.]], [[2., 6.], [6., 20.]]])
        // with tf.GradientTape() as tape:
        //    tape.watch(x)
        //    l = tf.reduce_sum(tf.linalg.cholesky(x))
        // print(tape.gradient(l, x))
```

Reply:
Sure.